config/crd/bases/anywhere.eks.amazonaws.com_tinkerbelldatacenterconfigs.yaml

```diff
@@ -35,21 +35,25 @@ spec:
           metadata:
             type: object
           spec:
-            description: "TinkerbellDatacenterConfigSpec defines the desired state
-              of TinkerbellDatacenterConfig \n Important: Run \"make generate\" to
-              regenerate code after modifying this file"
+            description: TinkerbellDatacenterConfigSpec defines the desired state
+              of TinkerbellDatacenterConfig
             properties:
               hookImagesURLPath:
                 description: HookImagesURLPath can be used to override the default
-                  Hook images path to pull from a local server
+                  Hook images path to pull from a local server.
                 type: string
               osImageURL:
                 description: OSImageURL can be used to override the default OS image
-                  path to pull from a local server
+                  path to pull from a local server.
                 type: string
+              skipLBSetup:
```

Comment:
instead of acronym 'LB', 'LoadBalancer' would be clearer when looking at cluster config

Reply:
done